src/Windows/Avalonia.Win32/FramebufferManager.cs

```diff
@@ -25,7 +25,7 @@ public ILockedFramebuffer Lock()
             UnmanagedMethods.GetClientRect(_hwnd, out rc);
             var width = rc.right - rc.left;
             var height = rc.bottom - rc.top;
-            if (_fb == null || _fb.Width != width || _fb.Height != height)
+            if ((_fb == null || _fb.Width != width || _fb.Height != height) && width > 0 && height > 0)
```

Comment:
@kekekeks this is an attempt to fix the problem that AvalonaStudio was seeing when the window was maximized and `width`/`height` of 0 was passed to `WindowFramebuffer`. I'm not sure if it's the correct fix.